Codeinwp/neve
# feat: removed style and builder rollback #3952

### Summary

Removed the Style switcher and builder migration notice.

### Will affect the visual aspect of the product

NO

### Screenshots 
<details open>
    <summary>Style Control (Pic. 1)</summary>

![image](https://github.com/Codeinwp/neve/assets/23024731/694487af-29e6-4a25-aa02-5075399419f4)
![image](https://github.com/Codeinwp/neve/assets/23024731/7a8900a0-5700-40c5-8118-3f6b9facecf1)
</details>

<details>
    <summary>Rollback Notice (Pic. 2)</summary>

![image](https://github.com/Codeinwp/neve/assets/23024731/ae318ae0-6b29-42b8-8581-ae9837da11c7)
</details>

### Test instructions

1. On a fresh instance of Neve with Neve PRO
2. set the following theme mods to `true`
    - `neve_auto_migrated_to_new_skin`
    - `neve_had_old_skin`
    - `hfg_header_layout`
    - `neve_migrated_builders`
    You can use WP CLI `wp theme mod set <theme_mod_name> <theme_mod_value>`
3. Check that the Style control inside Customizer is not available anymore (see Pic. 1)
4. Check that the Rollback Notice for the Header is not present (see Pic. 2)
5. Check that the functionality is maintained when modifying the Header and that no errors are reported.

## Check before Pull Request is ready:



Closes #3952.
